crates/sui-bridge-indexer/src/migrations/2024-09-05-180103_governance_actions/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE governance_actions
+(
+    id                          BIGSERIAL    PRIMARY KEY,
+    nonce                       BIGINT,
+    data_source                 TEXT         NOT NULL,
+    txn_digest                  bytea        NOT NULL,
+    sender_address              bytea        NOT NULL,
+    timestamp_ms                BIGINT       NOT NULL,
+    action                      text         NOT NULL,
```

Comment:
same, avoid using string, there is BridgeActionType https://github.com/MystenLabs/sui/blob/main/crates/sui-bridge/src/types.rs#L197

Reply:
Are you suggesting we update the db schema to include a "BridgeActionType" enum? We aren't doing this with any of our other enums (for instance `status` and `data_source` are also saved as Text), but not a bad idea...

Reply:
yeah that's the suggestion. I think we should rework all of them to integer post launch